corehq/apps/reports/standard/monitoring.py

```diff
@@ -1744,6 +1744,11 @@ def _report_data(self):
             case_owners = case_owners.union((user.user_id, user.location_id))
             case_owners = case_owners.union(user.group_ids)
 
+        try:
+            case_owners.remove(None)
+        except KeyError:
+            pass
```

Comment:
This will only remove the first `None` from `case_owners`.  Assuming you're not concerned about there being other falsy values, you could do:
```python
case_owners = filter(None, case_owners)
```

Reply:
`case_owners` is a `set` so `remove` should be fine

Reply:
Oh right.  Carry on then.